model-optimizer/extensions/ops/ctc_greedy_decoder.py

```diff
@@ -32,13 +33,15 @@ def __init__(self, graph: Graph, attrs: dict):
             'reinterp_shape': True,
 
             'in_ports_count': 2,
-            'out_ports_count': 1
+            'out_ports_count': 1,
+
+            'ctc_merge_repeated': 1     # We use int value because we extract this attr as int
```

Comment:
I don't understand the comment. It doesn't really matter how we extract this value. The default value should be the valid one, so I suggest to use `True` value here.

Reply:
Comment removed, we can use `True ` value, fixed.